recipes/geographiclib/all/conanfile.py

```diff
@@ -1,9 +1,12 @@
 from conans import ConanFile, CMake, tools
 import os
 
+required_conan_version = ">=1.28.0"
```

Comment:
:exploding_head:

Reply:
https://docs.conan.io/en/latest/changelog.html#jul-2020

Reply:
Can I apply this to recipes that require 1.26 features? This feature is absolutely priceless. Can there be a hook for this?!

Reply:
Of course! But it only works for Conan >=1.28.0. If you use with an older version, Conan will ignore it. However, it's supported in conan.conf since 1.27 (https://github.com/conan-io/conan/pull/7183), but it's global.

Reply:
A hook won't be used, we think we should not force this feature, we want to see the community adherence first. If in the future we see more and more people we can add a hook, or course.